portal/src/graphql/portal/AuthenticationLoginIDSettings.tsx

```diff
@@ -285,6 +286,7 @@ const AuthenticationLoginIDSettings: React.FC<Props> = function AuthenticationLo
           initiallyExtended={usernameEnabled}
           extendable={true}
           readOnly={!usernameEnabled}
+          extendButtonAriaLabelId={"AuthenticationWidget.usernameExtend"}
```

Comment:
Since this component is owned by us, we can make it to support `ariaLabel: React.ReactNode`. Then we can pass it `<FormattedMessage>`

Reply:
However we need to pass the props to `IconButton` in fluentUI. The ariaLabel in `IconButton` must be string. @louischan-oursky

Reply:
Fair enough.